measurements/app.py

```diff
@@ -58,6 +78,7 @@ def create_app(*args, **kw):
     from measurements import views
 
     app = Flask(__name__)
+    app.json_encoder = FlaskJSONEncoder
```

Comment:
Does this mean that the thing you return from handlers is `JSON` encoded by flask using this filter?

(I assume that was already the case, but you had the issue that some objects were not trivially JSON encodeable, like datetime, and hence you needed to make sure they were explicitly in the code.)

Reply:
Yes, but I extend it with the serialisation of datetime to avoid duplicating code in the handlers.

Reply:
perfect